safeCode.mk

```
@@ -0,0 +1,141 @@
+#Code filessafe to build without the ASM processor.
+#Dont change this unless you know what you are doing.
+
+SAFE_OVERLAY_FILES :=	 \
```

```suggestion
SAFE_CODE_FILES :=	 \
```
? (this is safeCode.mk)